pylint/checkers/refactoring.py

```diff
@@ -18,6 +18,7 @@
 
 from pylint import interfaces
 from pylint import checkers
+from pylint.checkers.logging import is_method_call
```

Comment:
We should not import objects from other checkers, let's keep their namespace as private. In this case, we can move ``is_method_call`` in ``pylint.checkers.utils``, and use it from both files. Also, let's do it in a separate commit.